src/components/BrandConfigProvider/theme/ThemeProvider.tsx

```diff
@@ -23,7 +49,23 @@ interface Props {
 }
 
 export function ThemeProvider({ theme: customTheme, children }: Props) {
-  const theme = merge({}, defaultTheme, customTheme);
+  // TODO: When we decide to support a dark theme, add this to context/state
+  const isThemeDark = false;
+
+  const materialLight = merge({}, MD3LightTheme, baseLightTheme, customTheme);
+  const materialDark = merge({}, MD3DarkTheme, baseDarkTheme, customTheme);
+
+  const { LightTheme, DarkTheme } = adaptNavigationTheme({
+    reactNavigationLight,
+    reactNavigationDark,
+    materialLight,
+    materialDark,
+  });
+
+  const lightTheme = merge({}, LightTheme, materialLight);
+  const darkTheme = merge({}, DarkTheme, materialDark);
+
+  const theme = isThemeDark ? darkTheme : lightTheme;
```

Comment:
Can we move all the light/dark stuff to a separate branch/PR?

Reply:
Cool, I was able to remove most light/dark references.